pkg/operator/controller/externaldns/deployment_test.go

```diff
@@ -3125,10 +3142,13 @@ func TestEnsureExternalDNSDeployment(t *testing.T) {
 }
 
 func testDeployment() *appsv1.Deployment {
+	secretHashAnnotation := make(map[string]string)
+	secretHashAnnotation["external-dns-credentials-test"] = "f17bceb5a060e33473c68229903ef5c517d9a172"
```

Comment:
This annotation comes a bit out of nowhere. There is dedicated function for the test deployment with annotation, this function can be used when an annotation is needed.

Reply:
Resolved 7734cdd74f2384679279d20f0130bcc9d728d821